composer.json

```diff
@@ -74,6 +73,7 @@
         "visiosoft/decimal-field_type": "~2.1.0",
         "visiosoft/integer-field_type": "~2.1.0",
         "visiosoft/list-field_type": "*",
+        "visiosoft/system-module": "~1.1.0",
```

Comment:
why ?

Reply:
the anomaly system module is logging all the views, there is no option to disable it